protocol/dubbo/dubbo_invoker.go

```diff
@@ -87,7 +87,7 @@ func (di *DubboInvoker) Invoke(ctx context.Context, invocation protocol.Invocati
 		err    error
 		result protocol.RPCResult
 	)
-	if di.reqNum < 0 {
+	if atomic.LoadInt64(di.reqNum) < 0 {
```

Comment:
plg check the github error:

go/protocol/dubbo/dubbo_invoker.go:90:22: cannot use di.reqNum (variable of type int64) as *int64 value in argument to atomic.LoadInt64)]"